sass/main.scss

```diff
@@ -3,6 +3,7 @@
   src: url('fonts/Inter.ttf');
   font-display: swap;
 }
+@import 'parts/archive.scss';
```

Comment:
shouldn't it be `parts/_archive.scss`? (leading underscore)

Reply:
yessir, i'll correct it